test/unittests/test_helpers.cpp

```diff
@@ -27,27 +26,6 @@ static constexpr size_t optionalDataSize =
 static_assert(optionalDataSize >= sizeof(evmc_result_optional_storage),
               "evmc_result's optional data space is too small");
 
-
-TEST(helpers, fnv1a)
-{
-    const uint8_t text[] = {'E', 'V', 'M', 'C'};
-    const auto h = fnv1a(text, sizeof(text));
-    EXPECT_EQ(h, sizeof(size_t) == 8 ? 0x15e05d6d22fed89a : 0xffaa6a9a);
-}
-
-TEST(helpers, is_zero)
```

Comment:
I don't think tests should be removed with deprecation. We still need to ensure nobody breaks the deprecated features.

Reply:
That being said, I think the test removed in https://github.com/ethereum/evmc/commit/279c1cd6e75b69947f33b33a95348939d3c72206 should also be brought back.